docs/DeveloperGuide.adoc

```diff
@@ -279,12 +279,12 @@ Refer to the guide <<DevOps#, here>>.
 
 *Target user profile*:
 
-* has a need to manage a significant number of tasks
-* needs reminders to take breaks or move on to cover all their work
+* has a need to manage a large number of tasks
```

Comment:
is there an upper limit to 'large'? what is the capacity we are aiming for?

Reply:
I think we can keep it ambiguous for now.